spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1657,6 +1670,54 @@
             end
           end
         end
+
+        describe 'the profile has deactivation_reason set to encryption_error' do
+          context 'the enrollment passes proofing with USPS' do
+            let(:pending_enrollment) do
+              create(
+                :in_person_enrollment, :pending, :with_notification_phone_configuration
+              )
+            end
+
+            before do
+              allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+              stub_request_passed_proofing_results
+            end
+
```

Comment:
This is current behavior. The deactivation reason should actually not be overwritten to nil. This will be handled in https://cm-jira.usa.gov/browse/LG-14591